beta/src/pages/learn/index.md

```diff
@@ -4,716 +4,491 @@ title: Quick Start
+        }}
       />
-    </section>
+    </>
   );
 }
 ```
 
 ```css
-img { margin: 0 10px 10px 0; height: 90px; }
-.avatar { border-radius: 50%; }
+.avatar {
+  border-radius: 50%;
+}
+
+.large {
+  border: 4px solid gold;
+}
 ```
 
 </Sandpack>
 
-You might wonder why `className="avatar"` uses quotes but `src={imageUrl}` uses curly braces. In JSX, curly braces are like a ["window into JavaScript"](/learn/javascript-in-jsx-with-curly-braces). They let you run a bit of JavaScript right in your markup! So `src={imageUrl}` reads the `imageUrl` prop declared on the first line and passed from the parent `Gallery` component.
-
-In the above example, all the data was written directly in markup. However, you'll often want to keep it separately. Here, the data is kept in an array. In React, you use JavaScript functions like [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to [render lists](/learn/rendering-lists) of things.
+In the above example, `style={{ }}` is not a special syntax, but a regular object inside the JSX curlies. You can use the `style` attribute when your styles depend on JavaScript variables.
 
-<Sandpack>
+## Conditional rendering {/*conditional-rendering*/}
 
-```js App.js
-import { people } from './data.js';
-import Profile from './Profile.js';
+In React, there is no special syntax for writing conditions. Instead, you'll use the same techniques as you use when writing regular JavaScript code. For example, you can an [`if`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else) statement to conditionally include JSX:
 
-export default function Gallery() {
-  return (
-    <section>
-      <h1>Amazing scientists</h1>
-      {people.map(person => (
-        <Profile
-          key={person.id}
-          name={person.name}
-          imageId={person.imageId}
-        />
-      ))}
-    </section>
-  );
-}
-```
-
-```js Profile.js
-export default function Profile({ name, imageId }) {
-  const imageUrl = (
-    'https://i.imgur.com/' +
-    imageId +
-    's.jpg'
-  );
-  return (
-    <img
-      className="avatar"
-      src={imageUrl}
-      alt={name}
-    />
-  );
+```js
+let content;
+if (isLoggedIn) {
+  content = <AdminPanel />;
+} else {
+  content = <LoginForm />;
 }
+return (
+  <div>
+    {content}
+  </div>
+);
 ```
 
-```js data.js
-export const people = [{
-  id: 0,
-  name: 'Creola Katherine Johnson',
-  imageId: 'MK3eW3A'
-}, {
-  id: 1,
-  name: 'Mario José Molina-Pasquel Henríquez',
-  imageId: 'mynHUSa'
-}, {
-  id: 2,
-  name: 'Mohammad Abdus Salam',
-  imageId: 'bE7W1ji'
-}, {
-  id: 3,
-  name: 'Percy Lavon Julian',
-  imageId: 'IOjWm71'
-}, {
-  id: 4,
-  name: 'Subrahmanyan Chandrasekhar',
-  imageId: 'lrWQx8l'
-}];
-```
+If you prefer terser code, you can use the [conditional `?` operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator). Unlike `if`, it works inside JSX:
```

Comment:
**problem**: "terser" is not Simple English

**solution**: try "shorter", "more compact"?